copy-paste from gitlab (#1)
ome/cookiecutter-omero-webapp: {{cookiecutter.repo_name}}/README.md

OMERO.{{cookiecutter.webapp_name}}
=======================

> This repository contains an

Reviewer: ```suggestion
This repository contains a
```

[cookiecutter](https://cookiecutter.readthedocs.io/) generated example

> of a minimal omero webapp, derived from openmicroscopy [webapp examples](https://github.com/ome/omero-web-apps-examples).

Reviewer: ```suggestion
of a minimal OMERO webapp, derived from openmicroscopy [webapp examples](https://github.com/ome/omero-web-apps-examples).
```

> This template repository contains a minimal example of OMERO.web apps minimal example (no JavaScript dependencies or build steps).

Reviewer: ```suggestion
This repository contains a minimal OMERO.web app (no JavaScript dependencies or build steps).
```
